src/components/FeaturedProducts.jsx

```diff
@@ -0,0 +1,25 @@
+import featuredProductsData from '../mocks/en-us/featured-products.json';
+import { SectionTitle } from './styles/SectionTitle.styled';
+import { CardContainer } from './styles/CardContainer.styled';
+import ProductCard from './ProductCard';
+
+const FeaturedProducts = () => {
+  const featuredProducts = featuredProductsData.results;
+  return (
+    <section>
+      <SectionTitle>Featured Products</SectionTitle>
+      <CardContainer>
+        {featuredProducts.map((featuredProduct) => (
+          <ProductCard
+            key={featuredProduct.id}
+            name={featuredProduct.data.name}
+            image={featuredProduct.data.mainimage.url}
+            buttonText="Go to product"
+          />
+        ))}
```

Comment:
I'm thinking this can be a component on its own, like a CardsLayout or something similar. Maybe you can reuse it on other places, where you need a map of ProductCards